ando/tools/generator/tests/test_nwb2bidsgenerator.py

```diff
@@ -3,9 +3,13 @@
 from pathlib import Path
 import re
 from datalad.api import install, Dataset
+import os
+ON_GITHUB = 'GITHUB_WORKFLOW' in os.environ
 
 from ando.tools.generator.nwb2bidsgenerator import NwbToBIDS, is_valid
 
+
+@unittest.skipIf(ON_GITHUB, 'Not running NWB test on github for performance reasons')
```

Comment:
@Slowblitz Did you confirm this test is only skipped on github and still executed locally?

Reply:
yes still done localy